### PR TITLE
Chore: Refactor data fixtures

### PR DESCRIPTION
**Pull request by andrea-williams**
Changes made in this PR:

- Organized data fixtures into /dummy and /real; 
- removed NaicsCategory model (based on data provided from business area and discussion with Dylan, we don't think we'll need the NaicsCategory table nor the `ciip_sector` field in the NaicsCode table); 
- added NaicsCode data (provided by business area)

Once we're releasing to prod, we'll be able to populate our prod database using the data fixtures in the /real directory.
Both the /real and /dummy fixture folders can be used in local development.

### CODE REVIEW

File: bc_obps/registration/api/naics.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
 from .api_base import router
 from typing import List
-from registration.models import NaicsCode, NaicsCategory
+from registration.models import NaicsCode
```
Comment (BCerki) [inline bc_obps/registration/api/naics.py:3]: yay deleting stuff :cartwheeling:

File: bc_obps/registration/fixtures/real/naicsCode.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    }
+  },
+  {
+    "model": "registration.naicsCode",
+    "pk": 2,
+    "fields": {
+      "id": 2,
+      "naics_code": "212114",
+      "naics_description": "Bituminous coal mining"
+    }
```
Comment (BCerki) [inline bc_obps/registration/fixtures/real/naicsCode.json:18]: these two are the only codes with `pk` and `id`, and all the other ones seem to have gone into the db fine, so it looks like we don't have to specify these manually

File: bc_obps/registration/fixtures/real/naicsCode.json [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  },
+  {
+    "model": "registration.naicsCode",
+    "fields": {
+      "naics_code": "212220",
```
Comment (BCerki) [inline bc_obps/registration/fixtures/real/naicsCode.json:23]: FYI, the naics code dropdown in the form shows the ids, not the code or the code title. We've got a ticket to address that already: https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/318

File: bc_obps/registration/fixtures/dummy/document.json [hunk 4]
```diff
@@ -1,24 +1,10 @@
 [
```
Comment (BCerki) [inline bc_obps/registration/fixtures/dummy/document.json:1]: can we please use something other than `dummy`? `mock` or some other suggestions here: https://www.khronos.org/about/inclusive-language#dummy

**Review by BCerki — COMMENTED** [review]
:cartwheeling: :cartwheeling: :cartwheeling:

File: bc_obps/registration/fixtures/dummy/document.json (hunk 4 above)
Comment (andrea-williams) [reply bc_obps/registration/fixtures/dummy/document.json:1]: Yikes. I'm very sorry, I didn't realize that "dummy" is an exclusionary term. Thanks for letting me know!
I've replaced it with `mock`

**Pull request closed without merging**